contentcuration/contentcuration/frontend/channelEdit/store.js

```diff
@@ -36,7 +36,11 @@ export const STORE_CONFIG = {
   actions,
   mutations,
   getters,
-  plugins: [DraggablePlugin, persistFactory('channelEdit', ['SET_VIEW_MODE'])],
+  plugins: [
+    DraggablePlugin,
+    persistFactory('channelEdit', ['SET_VIEW_MODE'], false),
+    ...clipboard.plugins,
```

Comment:
Apparently, plugins do get activated when they're added to modules, so this ensures the clipboard plugins are loaded for the channel edit page Vuex store.

Reply:
Interesting, good catch